ts/features/payments/common/types/PaymentAnalytics.ts

```diff
@@ -22,6 +22,7 @@ export type PaymentAnalyticsData = {
   transactionsHomeLength?: number;
   receiptUser?: PaymentsAnalyticsReceiptUser;
   receiptFirstTimeOpening?: boolean;
+  receiptFirstTimePDF?: boolean;
```

Comment:
```suggestion
  receiptFirstTimeOpeningPDF?: boolean;
```
What do you think about renaming this property to be more understandable?

Reply:
Addressed here https://github.com/pagopa/io-app/pull/6797/commits/9b561f89d3c789a306ca8b0f1db9c68d214b0a00